restservice/src/main/java/org/itevents/dao/mybatis/sql_session_dao/AbstractMyBatisDao.java

```diff
@@ -16,7 +17,11 @@
     private SqlSessionFactoryBean sqlSessionFactoryBean;
 
     @PostConstruct
-    public void init() throws Exception {
-        setSqlSessionFactory(sqlSessionFactoryBean.getObject());
+    public void init() {
+        try {
+            setSqlSessionFactory(sqlSessionFactoryBean.getObject());
+        } catch (Exception e) {
+            throw new DaoException(e.getMessage(), e);
```

Comment:
@romach if this exception wil happen, this will cause error 500  because of unusual problem in DAO layer,, also this exception not for our Controller layer.